scripts/submit-analytics.js

```diff
@@ -118,8 +118,25 @@ function isAnalyticsDisabled() {
 }
 
 function getRealmVersion() {
-  const packageJson = fse.readJsonSync([getProjectRoot(), "node_modules", "realm", "package.json"].join(path.sep));
-  return packageJson["version"];
+  let version = "unknown";
+  let packageJsonPath = "";
+  try {
+    packageJsonPath = [getProjectRoot(), "package.json"].join(path.sep);
+    const packageJson = fse.readJsonSync(packageJsonPath);
+    if (packageJson["name"] === "realm") {
+      version = packageJson["version"];
```

Comment:
This will simply fall through and report "unknown" if no error is thrown?

I suggest [relying on the `npm_config_local_prefix`](https://github.com/realm/realm-js/pull/4163#discussion_r792568357). If that directory's package.json has "realm" as its name, simply skip submitting analytics all together.

Reply:
It is using `npm_config_local_prefix` through `getProjectRoot()`. But in some cases (in `integration-tests` if I remember correctly), it is not set.

 > If that directory's package.json has "realm" as its name, simply skip submitting analytics all together.

It makes sense to add this to `isAnalyticsDisabled()`.